pkgs/development/libraries/avro-c++/default.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation {
     description = "A C++ library which implements parts of the Avro Specification";
     homepage = "https://avro.apache.org/";
     license = stdenv.lib.licenses.asl20;
```

Comment:
```suggestion
    license = lib.licenses.asl20;
```

Reply:
This won't eval as `lib` is not in scope.

Reply:
Then please add lib to inputs. This is a recent contributing guide change and stdenv.lib is deprecated.